template/Configuration.mustache

```diff
@@ -1,365 +1,334 @@
-{{>partial_header}}
-{{^net35}}
-    using System.Collections.Concurrent;
-{{/net35}}
+/*
+ * Fingerprint Pro Server API
+ *
+ * Fingerprint Pro Server API allows you to get information about visitors and about individual events in a server environment. It can be used for data exports, decision-making, and data analysis scenarios. Server API is intended for server-side usage, it's not intended to be used from the client side, whether it's a browser or a mobile device.
```

Comment:
Using hardcoded description instead of {{>partial_header}} causes that if description will be updated in schema it will not be here

Reply:
Fixed